argmin-math/src/vec/shur.rs

```diff
@@ -0,0 +1,382 @@
+// Copyright 2018-2022 argmin developers
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+// http://opensource.org/licenses/MIT>, at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+
+#![allow(non_snake_case)]
+use crate::vec::HessenbergTransformer;
+
+const SHUR_MAX_ITERATIONS: usize = 100;
+
+/// Transforms a matrix into a Shur form.
+/// This transformation is an intermediate step to Eigendecomposition
+pub(crate) struct ShurTransformer<T> {
```

Comment:
Isn't it "Schur" instead of "Shur"? Maybe I'm thinking of something different.

Reply:
I think you are correct, it should be `Schur`